components/Main/MuscleGroups/MuscleGroupItem/MuscleGroupItem.js

```diff
@@ -0,0 +1,24 @@
+import Image from "next/image";
+import Link from "next/link";
+import { StyledMuscleGroupItem } from "./StyledMuscleGroupItem";
+
+const MuscleGroupItem = ({ muscle }) => {
+  return (
+    <StyledMuscleGroupItem>
+      <Link href={`/exercises/${muscle}`}>
+        <li>
+          <Image
+            src={`/images/${muscle}.svg`}
+            width={158}
+            height={80}
+            alt="muscle"
```

Comment:
I guess this is a special muscle, maybe use the full muscle name her ? `{ muscle }`

Reply:
Thanks ! :)